Dockerfile

```diff
@@ -28,18 +28,30 @@ ARG STRIPE_API_KEY
 ARG STRIPE_WEBHOOK_SIGNING_SECRET
 ARG PHX_SERVER
 ARG DATABASE_URL
+ARG PLATFORM=amd64
 
-ARG BUILDER_IMAGE="hexpm/elixir:${ELIXIR_VERSION}-erlang-${OTP_VERSION}-debian-${DEBIAN_VERSION}"
+ARG BUILDER_IMAGE="hexpm/elixir-${PLATFORM}:${ELIXIR_VERSION}-erlang-${OTP_VERSION}-debian-${DEBIAN_VERSION}"
 ARG RUNNER_IMAGE="debian:${DEBIAN_VERSION}"
 
-FROM --platform=linux/amd64 ${BUILDER_IMAGE} as builder
+FROM --platform=linux/${PLATFORM} ${BUILDER_IMAGE} as builder_amd64
+ONBUILD RUN wget -O - 'https://github.com/elm/compiler/releases/download/0.19.1/binary-for-linux-64-bit.gz' | gunzip -c >/usr/local/bin/elm
```

Comment:
Can the version be a variable ?

Reply:
Would be cleaner, but Elm don't evolve a lot, this is the last version since 2019 https://github.com/elm/compiler/releases ^^

Reply:
Oh I see